sdk/tools/mkisofs/schilytools/include/schily/wctype.h

```diff
@@ -46,7 +46,7 @@
 
 #if	defined(HAVE_ISWPRINT) && defined(USE_WCHAR)
 #ifndef	USE_WCTYPE
-#undef	USE_WCTYPE
+#undef	USE_WCTYPE //FIXME: bug! This is #define USE_WCTYPE in newer versions, see https://github.com/roytam1/schilytools/tree/master/include/schily/wctype.h for surrounding changes also
```

Comment:
See especially
https://github.com/roytam1/schilytools/commit/344d1e2be1b761110b54b3f52899c02faf901bc9#diff-9e4c9cd5c78fb8520b73d074451cac9e10b761153c47305b06b15ecdcefa1510
and
https://github.com/roytam1/schilytools/commit/7a8be1fb62480f0937591189008608a016b2ef66#diff-9e4c9cd5c78fb8520b73d074451cac9e10b761153c47305b06b15ecdcefa1510